src/app/expense/add-edit-expense/add-edit-expense.component.scss

```diff
@@ -1,96 +1,56 @@
-@import '/src/app/layout/color.scss';
-// Expense Container design
-
-$font-size-small: 14px;
-$font-size-medium: 16px;
-
-@import '/src/app/layout/color.scss';
-
-// Expense Container design
-.expense-form-container {
-  background-color: $white;
-  padding: 20px;
-  box-shadow: 0px 4px 8px $black;
-  overflow-x: hidden;
-//Form group design
-  .form-group {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    align-items: center;
-    margin-bottom: 10px;
-//Form labbel design
-    label {
-      flex: 1;
-      margin-right: 10px;
-      font-weight: bold;
-    }
-//Form input field design
-    input[type="text"],
-    input[type="number"],
-    select,
-    textarea {
-      flex: 2;
-      height: 30px;
-      border: 1px solid $black;
-      border-radius: 4px;
-      padding: 5px;
-      font-size: $font-size-medium;
-      box-sizing: border-box;
-      margin-top: 5px;
-
-      &:focus {
-        outline: none;
-        border-color: $black;
-      }
-    }
-
-    select[multiple] {
-      height: auto;
-    }
-//Form  error message design 
-    .error-msg {
-      padding: 10px 5px;
-      color: $warning;
-      font-size: $font-size-small;
-      margin-top: 5px;
-      flex: 2;
-    }
-  }
-//Split option design
-  .split-options {
-    display: flex;
-    flex-direction: column;
-
-    label {
-      margin-right: 20px;
-    }
-  }
-//Button design
-  button[type="submit"] {
-    background-color: $green;
-    border: none;
-    color: $white;
-    font-size: $font-size-medium;
-    padding: 10px 20px;
-    border-radius: 4px;
-    cursor: pointer;
-    transition: background-color 0.3s ease;
-
-    &:hover {
-      color: $green;
-      background-color: $white;
-    }
-
-    &:disabled {
-      background-color: $light-gray;
-      cursor: not-allowed;
-    }
-  }
+import { Component } from '@angular/core';
```

Comment:
Why ts file code in .scss file?

Reply:
Sorry Di mistakenly I put that code on the scss I will remove that code before demo